src/pe-simple-header/widget/pe-simple-header.u-client-script.js

```diff
@@ -0,0 +1,233 @@
+function($rootScope, $scope, $window, $location,$anchorScroll, $interval, $timeout, $uibModal, $sce, snRecordWatcher, cabrillo, spUtil, spAriaUtil) {
+/* widget controller */
```

Comment:
- [ ] please remove 
1. $interval 
2. $sce
3. snRecordWatcher 

Above items are not used. thx

Reply:
Removed